Assets/Scripts/NpcBehaviors/SampleNpc.cs

```diff
@@ -38,6 +40,25 @@ private void Update()
             Interact(2);
     }
 
+    // Using this to test if inventory integration works
+    public override void CollectedItem(InventoryItemData item, int quantity)
+    {
+        base.CollectedItem(item, quantity);
+    }
+
+    // Returns dialogue based on if player has bypass item
+    protected override int ChooseDialoguePath(PlayerResponse option)
```

Comment:
Can you add a warning if the options are out of range? Just to make debugging a little easier if any problems come up.